src/oemof_tabular_plugins/wefe/facades/functions.py

```diff
@@ -0,0 +1,996 @@
+        cumulative temperature already experienced by the plant [K]
+    delta_i50b: numeric
+        hourly increase in i50b
+    t_sum: numeric
+        cumulative temperature until maturity [K]
+    i50a: numeric
+        cumulative temp. from sowing on to reach 50% of
+        solar radiation interception during growth [K]
+    i50b: numeric
+        cumulative temp. until maturity to fall back to 50% of
+        solar rad. intercept. during decline [K]
+    f_solar_max:
+        maximum solar radiation interception [-]
+
+    Returns
+    ----------
+    f_solar: numeric, in [0,f_solar_max]
+        solar radiation interception factor for biomass generation [-]
+
+    Notes
+    ----------
+    SIMPLE presents two equations for f_solar = f(cum_temp), one for leaf growth and
+    one for leaf senescence, but fails to specify the transition point (a cum_temp value).
+    To get a smoove transition, the transition point was defined as
+    'cum_temp_to_reach_f_solar_max' and calculated by setting the first equation for f_solar
+    equal to 0.999f_solar_max and solving for cum_temp.
+
+    The delta_i50b values (function 'delta_i50b') are cumulated and the cumulative value
+    is added to i50b for every time step so that i50b is increasing over time as intended.
+    """
+    cum_temp_to_reach_f_solar_max = i50a - 100 * np.log(1 / 999)
+    if cum_temp < 1:
+        f_solar = 0
+    # First equation: Leaf growth period
+    elif cum_temp < cum_temp_to_reach_f_solar_max:
+        i50b += delta_i50b
+        f_solar = f_solar_max / (1 + np.exp(-0.01 * (cum_temp - i50a)))
+    # Second equation: Leaf senescence period
+    elif cum_temp < t_sum:
+        i50b += delta_i50b
+        f_solar = f_solar_max / (1 + np.exp(0.01 * (cum_temp - (t_sum - i50b))))
+    else:
+        f_solar = 0
+    return f_solar
+
+
+def soil_heat_flux(ghi, r_n):
+    """
+    FAO56:
+    Soil heat flux for hourly calculations
+    Eq. (45) and (46)
+
+    Parameters
+    ----------
+    ghi: numeric
+        gloabl horizontal irradiation [W/m²]
+    r_n: numeric
+        net radiation [W/m²]
+
+    Returns
+    ----------
+    g: numeric
+        soil heat flux at that time step (hourly) [W/m²]
+
+    Notes
+    ----------
+    Night time is simply specified at the time when ghi is below 1.
+    R_n should be positive during the day and negative during the night.
+    However, errors will be insignificant.
+    """
+    ghi_at_night = 1
+    if ghi > ghi_at_night:
+        g = 0.1 * r_n
+    else:
+        g = 0.5 * r_n
+    return g
+
+
+def vap_pressure(t):
+    """
+    FAO56:
+    Water vapour saturation pressure at specific temperature
+    Eq. (11)
+
+    Parameters
+    ----------
+    t: numeric
+        tempearture [°C]
+
+    Returns
+    ----------
+    e: numeric
+        saturation vapour pressure [kPa]
+    """
+    e = 0.6108 * np.exp(17.27 * t / (t + 237.3))
+    return e
+
+
+def potential_evapotranspiration(z, t_air, t_dp, w10, r_n, g, **kwargs):
```

Comment:
this has way to many magic numbers to my taste, it makes it impossible to do unit check because we don't know what's behind the number